src/components/game/component.jsx

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
+import { Container, Button } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 
 import Board from '../board';
 
-const Game = ({ history, moveNumber }) => {
+const Game = ({ history, moveNumber, onClick, jumpTo }) => {
   const current = history[moveNumber];
+  const moveMap = history.map((eachMove, move) => {
+    const moveHistory = move ? `Go to move #${move}` : 'Restart';
+    return (
+      <Button
+        size="sm"
+        variant="outline-dark"
+        key={[move]}
+        onClick={() => jumpTo(move)}
+      >
+        {moveHistory}
+      </Button>
+    );
+  });
```

Comment:
This block of code can move outside of the component. I'd prefer returning an object rather than returning JSX out of the function. We can then run the map functionality within the render portion of the function.

Having the object outside of the component will make unit testing this component much easier.

Reply:
Cool will do